core-bundle/src/Resources/contao/library/Contao/Image.php

```diff
@@ -686,8 +688,22 @@ public static function getHtml($src, $alt='', $attributes='')
 
 		if (!is_file($rootDir . '/' . $src))
 		{
+			try
+			{
+				$deferredImage = $container->get('contao.image.image_factory')->create($rootDir . '/' . $src);
+			}
+			catch (\Exception $e)
+			{
+				$deferredImage = null;
+			}
+
+			// Handle deferred images
+			if ($deferredImage instanceof DeferredImageInterface)
+			{
+				// ignore
```

Comment:
Shouldn't *ignore* mean `return ''`?

Reply:
No, it should be ignored that the image doesn’t exist (yet) because it is a deferred image.

Reply:
well the if-statement does not ignore anything, it just continues the operation?

Reply:
Yes it should continue the operation even though the image does not exist.
Do you think we should rephrase the comment to `// continue`?

Reply:
What about `(!$deferredImage instanceof DeferredImageInterface) && file_exists($rootDir . '/' . $webDir . '/' . $src)`?

Reply:
It would then use the `else` branch and would `return ''` which is not what we want in this case.

Reply:
```php
if (file_exists($rootDir . '/' . $webDir . '/' . $src)) {
    $src = $webDir . '/' . $src;
} elseif (!$deferredImage instanceof DeferredImageInterface) {
    return '';
}
```

Reply:
4fa47a7ce9b2628be0cba6c44a490323e9f041a3